hr_payslip_ytd_amount/README.rst

```diff
@@ -0,0 +1,72 @@
+Payslip Year-to-date Amount
+===========================
+
+This module implements a payslip model with a year-to-date amount column.
+The slip is rendered in the employee's language.
+
+
+Installation
+============
+
+ - Nothing to do except install the module
+
+
+Configuration
+=============
+
+Translate the names of your salary rules that appear on payslips.
+One way to do this is to create a .py file with the list of terms to translate.
+
+example:
+
+from openerp.tools.translate import _
+
+[
+	_('Gross Salary'),
+	_('Net'),
+	_('Income tax'),
+]
+
+Then the terms will appear in your pot file next time you generate it.
+
+
+Usage
+=====
+
+Go to Human Resources -> Payroll -> Employee Payslips
+Select the payslips you wish to generate, then click on Print -> Employee PaySlip (YTD Amount)
+
+If you wish to translate the payslip in the employee's language, you need to assign him to a user.
+Otherwise, the language of the user requesting the report will be used.
+
```

Comment:
Hi @eLBati , thanks for the contribution, could you please add the Try me on Runbot section